tests/lib/util.py

```diff
@@ -147,7 +147,7 @@ def create_pyproject_toml(
             """
         ),
         commit=False,
-        **kwargs
+        **kwargs,
```

Comment:
Please undo

Reply:
mmm I didn't author the changes on this file. This was done via pre-commit ci :thinking: https://github.com/dolfinus/setuptools-git-versioning/pull/50/commits/8d90b979554821e3b670d7300a58ee370fb81efe

Reply:
Do you mind if I just drop this commit and force push?